site/content/resources/misc/your-hr-tech-stack-blueprints-for-success/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Your HR Tech Stack: Blueprints for Success"
+date: "May 19, 2020"
+---
+{{< img src="your-hr-tech-stack-blueprints-for-success.jpg" alt="" >}}
+
+A blueprint of your HR tech stack is your source of truth for yourself as well as all your organization.  It is really just a detailed picture of all the processes and technology used by your organization for all HR functions from sourcing to onboarding and career development.  It is a clear picture of what tools your organization has.  It’s a communication tool to evaluate the current state of HR processes within your organization. If it’s a mess, then your results will be a mess.  How do you optimize and enhance your process without knowing what you have?  How do you strengthen the total effectiveness of your organization when your not sure who uses what tools to do what?  Here are **five steps** to help you create a cohesive blueprint of your HR tech stack for your organization.
+
+**Step One:**  Put together a complete inventory.  Identify all the systems you have, every piece of HR software your organization uses.  This may sound easy but most organizations don’t even know what they have in their HR tech stack.  They don’t know who uses what, or why they use it.  You will probably find software that your organization pays for but that no one uses or even knows about.  This might require you to go from department to department and see what they each use to get a big picture and a complete inventory.  Go to procurement and see what contracts are on file.  Go to finance, and see what your organization is paying for.  Another important part of this process is to observe what people are doing.  HR functions and HR tech will usually touch every aspect of your organization from the C-suite all the way down to a new-hire.  People may not know that they are actually using HR tech for an essential part of their job function.  This will take the most time and it may require a large amount of detective work, but you won’t know what tools and processes you have in your inventory until you complete this vital step.  
```

Comment:
```suggestion
 **Step One:**  Put together a complete inventory.  Identify all the systems you have, every piece of HR software your organization uses.  This may sound easy but most organizations don’t even know what they have in their HR tech stack.  They don’t know who uses what, or why they use it.  You will probably find software that your organization pays for but that no one uses or even knows about.  This might require you to go from department to department and see what they each use to get a big picture and a complete inventory.  Go to procurement and see what contracts are on file.  Go to finance, and see what your organization is paying for.  Another important part of this process is to observe what people are doing.  HR functions and HR tech will usually touch every aspect of your organization from the C-suite all the way down to a new-hire.  People may not know that they are actually using HR tech for an essential part of their job function.  This will take the most time and it may require a large amount of detective work, but you won’t know what tools and processes you have in your inventory until you complete this vital step.  
```